src/index.mako.html

```diff
@@ -7,7 +7,7 @@
 % endif
 >
   <head>
-    <!--![if !HTML5]>
+    <!--[if !HTML5]>
```

Comment:
It is. htmlmin (python) required this commenting format to preserve the comment after minification while html-minifier preserved them by default.